src/forms/assetForm.tsx

```diff
@@ -253,15 +253,17 @@ class AssetForm extends React.Component<AssetFormProps, AssetFormState> {
       });
   }
   getAssetNumber() {
-    axios
-      .get(API_ROOT + "api/assets/asset-number", getHeaders(this.props.token))
-      .then((res: any) => {
-        this.setState({
-          values: updateObject(this.state.values, {
-            asset_number: res.data.asset_number,
-          }),
-        });
-      });
+    if(!this.props.changePlan) {
```

Comment:
just realized how full stack i became in this ev. i was more excited to review ur front end changes than i was to review ur backend changes

Reply:
@benauriemma put this comment in ur presentation pleaees